src/config.rs

```diff
@@ -115,3 +119,48 @@ impl Default for OpCosts {
         Self::new()
     }
 }
+
+#[allow(missing_docs)]
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct HostCosts {
+    pub block_height: Gas,
+    pub callee: Gas,
+    pub caller: Gas,
+    pub debug: Gas,
+    pub emit: Gas,
+    pub gas_consumed: Gas,
+    pub gas_left: Gas,
+    pub panic: Gas,
+    pub get: Gas,
+    pub put: Gas,
+    pub hash: Gas,
+    pub query: Gas,
+    pub transact: Gas,
+}
+
+impl HostCosts {
```

Comment:
nit, but in general new should be defined in terms of default, not the other way around.

Reply:
Why is that? Especially since `new` is `const` and `default` isn't.